packages/accounts-2fa/2fa-server.js

```diff
@@ -0,0 +1,95 @@
+import { Accounts } from 'meteor/accounts-base';
+import twofactor from "node-2fa";
+import QRCode from "qrcode-svg";
+import { Meteor } from "meteor/meteor";
+
+Accounts.checkUserHas2FAEnabled = selector => {
+  if (!Meteor.isServer) {
+    throw new Meteor.Error(400, "The function checkUserHas2FAEnabled can only be called on the server");
+  }
+
+  if (typeof selector === 'string') {
+    if (!selector.includes('@')) {
+      selector = {username: selector};
+    } else {
+      selector = {email: selector};
+    }
+  }
+
+  const user = Meteor.users.findOne(selector) || {};
```

Comment:
I think we have some other ways to specify a custom selector in `accounts-password`. Should we support the same here? I'm not sure.